### PR TITLE
DKB-Trace 2.2 raytracer 

### PR DESCRIPTION
**Pull request by digitalbox94**
_Motivation: Provide the DKB-Trace raytracer available for Synology with a PHP Front-End access. Original project from David K. Buck.

### Checklist
- [] Build rule `all-supported` completed successfully
- [] Package upgrade completed successfully
- [] New installation of package completed successfully

### CODE REVIEW

File: cross/dkb-trace/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PKG_NAME = dkb-trace
+PKG_VERS = 2.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://digitalbox.chez.com/sources
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = 
+
+HOMEPAGE = http://digitalbox.chez.com/
```
Comment (publicarray) [inline cross/dkb-trace/Makefile]: This field is for the project homepage, helpurl. Example: http://aminet.net/package/gfx/3d/DKBTrace

File: cross/dkb-trace/Makefile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+PKG_NAME = dkb-trace
+PKG_VERS = 2.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://digitalbox.chez.com/sources
```
Comment (publicarray) [inline cross/dkb-trace/Makefile]: Personalty I prefer a more authoritative source. chez(dot)com is listed as engaged in malware distribution: https://hosts-file.net/default.asp?s=chez.com

File: cross/dkb-trace/Makefile [hunk 3]
```diff
@@ -0,0 +1,19 @@
+PKG_NAME = dkb-trace
```
Comment (publicarray) [inline cross/dkb-trace/Makefile]: Why not use the maintained https://github.com/POV-Ray/povray? As far as I understand it is based from the dkb-trace source.

File: spk/dkb-trace/Makefile [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+DEPENDS  = cross/$(SPK_NAME)
+
+MAINTAINER = DigitalBox
+MAINTAINER_URL = "https://digitalboxweb.wordpress.com/"
+
+DESCRIPTION = "DKB-Trace RayTracer is a raytracing tool written by David K. Buck. This is the ancestor of POV-Ray and was written completely in C. It is a photo-realistic 3-D object rendering shader, suitable for realizing the output of CAD programs or generally wasting LOTS of CPU time drawing pretty pictures.  It features a command-line interface and reads plain text input files in an easy to use and understand structured scene description language.  It writes output files in several popular 24-bit image formats, including DKB/QRT Raw and Truevision Targa-24 formats. Original website : ftp://ftp.ccs.carleton.ca/pub/dkbtrace/dkb2.12/ . Synology port : https://digitalboxweb.wordpress.com/"
+DISPLAY_NAME = DKB_Trace
+CHANGELOG = Version 2.2 with web access
+
+HOMEPAGE   = https://digitalboxweb.wordpress.com/
```
Comment (publicarray) [inline spk/dkb-trace/Makefile]: Ditto

File: cross/dkb-trace/Makefile (hunk 1 above)
Comment (digitalbox94) [reply cross/dkb-trace/Makefile]: Ok, indeed I can add the my GitHub homepage instead

File: cross/dkb-trace/Makefile (hunk 3 above)
Comment (digitalbox94) [reply cross/dkb-trace/Makefile]: POVRay is more complex to port, but I would be happy if someone is interested by this one

File: spk/dkb-trace/Makefile (hunk 4 above)
Comment (digitalbox94) [reply spk/dkb-trace/Makefile]: Agree for project GitHub link also

File: cross/dkb-trace/Makefile (hunk 3 above)
Comment (digitalbox94) [reply cross/dkb-trace/Makefile]: I've successfully compiled POVRay for braswell, but currently it requires some tricks like without-openexr, and some manual changes in configure before automating the configure/make process.
But most of the hard work is completed for the cross part :)

**Pull request closed without merging**